server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -768,7 +769,7 @@ public void setMergeState(MergeInfo info, MergeState state)
         try {
           info.write(out);
         } catch (IOException ex) {
-          throw new RuntimeException("Unlikely", ex);
+          throw new UncheckedIOException("Unlikely", ex);
```

Comment:
It looks like this is impossible, based on the implementation. If so, this should be AssertionError, which represents the programmer's assertion of it not being possible, on the assumption that the code is written correctly.